app/src/main/java/com/isc/hermes/controller/IncidentsGetterController.java

```diff
@@ -39,27 +43,34 @@ public IncidentsGetterController() {
         this.incidentsDataProcessor = IncidentsDataProcessor.getInstance();
         this.incidentGetterModel = new IncidentGetterModel();
         this.executorService = Executors.newSingleThreadExecutor();
+        mapboxMap = MapManager.getInstance().getMapboxMap();
     }
 
     /**
      * Retrieves incidentsPointView near the specified location within a radius.
      *
-     * @param mapboxMap The Mapbox map instance.
      */
-    public void getNearIncidentsWithinRadius(MapboxMap mapboxMap, Context context) {
+    public void getNearIncidentsWithinRadius(Context context) {
         incidentsPointView = IncidentPointVisualizationController.getInstance(mapboxMap, context);
+        //mapboxMap.addOnCameraIdleListener(() -> loadNearIncidents(mapboxMap, context));
```

Comment:
Remove commented code

Reply:
I'll remove it, I was waiting for the decision if we will continue with the lazy loading or not.